ui/types/vault/app-types.ts

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) HashiCorp, Inc.
  * SPDX-License-Identifier: BUSL-1.1
  */
-import type EmberDataModel from '@ember-data/model';
+import type EmberDataModel from 'ember-data/model'; // eslint-disable-line ember/use-ember-data-rfc-395-imports
```

Comment:
A comment might be helpful about why we're not importing from `@ember-data/model`

Reply:
Just a heads up if someone were to try and change it `ts` would holler back.
![Screenshot 2025-02-25 at 3 26 07 PM](https://github.com/user-attachments/assets/a8daca09-91d0-405b-9990-abc571ba6677)

Reply:
Makes sense!